.circleci/config.yml

```diff
@@ -10,23 +10,7 @@ orbs:
 # https://hub.docker.com/r/cimg/node/tags/
 
 jobs:
-  install-and-cibuild-node12: # cibuild using node 12 & npm 6
```

Comment:
We need to bring these node 12 test back possibly by adding the revised package of regl to the repo.

Reply:
Do those tests actually break now, or was removing them just a part of troubleshooting the node version?

Reply:
This bit is omitted by mistake during my "restore" from troubleshooting where it was removed.